hedera-mirror-monitor/src/main/java/com/hedera/mirror/monitor/publish/PublishMetrics.java

```diff
@@ -64,14 +64,16 @@
     private final AtomicLong lastCount = new AtomicLong();
     private final AtomicLong lastElapsed = new AtomicLong();
 
-    public void onSuccess(PublishRequest request, PublishResponse response) {
+    public void onSuccess(PublishResponse response) {
         counter.incrementAndGet();
-        recordMetric(request, response, SUCCESS);
+        recordMetric(response.getRequest(), response, SUCCESS);
     }
 
-    public void onError(PublishRequest request, Throwable throwable) {
+    public void onError(PublishException publishException) {
+        PublishRequest request = publishException.getPublishRequest();
         String status;
         TransactionType type = request.getType();
+        Throwable throwable = publishException.getCause() != null ? publishException.getCause() : publishException;
```

Comment:
nit: same here, better to go for positive scenario first
```suggestion
        Throwable throwable = publishException.getCause() == null ? publishException : publishException.getCause();
```